### DISCUSSION TITLE
Software Screengrabber on Arch Based System (Manjaro)

### DISCUSSION THREAD

**Discussion by epictechlab** [awawa-dev/HyperHDR#533] [General]
HI there,

I've switched from Hyperion-ng to HyperHDR on my Linux maschine, because i have it running on my TV and it gaves me much better results in the color Quality. 

In Hyperion.ng I enabled Software Screengrabber and select X11 an the monitor i wanted to screencapture. 
In HyperHDR I cant find these settings. 
I've disabled systemctl start in the console, and started HyperHDr manually from the applications menu. I Can change color Effects and can see the Startup effect like Rainbow Swirl. 

In the software screencapture menu is only Automatic and something "pipewire" (i cant remember the correct naming) showing.
Is there something i cant See?

Thanks for the great tool, its awesome (on My TV)

best regarts

**Comment by awawa-dev** [comment]
Hi
HyperHDR automatically uses the best software screen capture method. Usually it's a pipewire/portal pipeline present almost on all modern architectures that enables to capture e.g. the Wayland sessions. It should also allows to capture X11 sessions but since X11 project currently has maintance-only status, its support is not the main goal of the pipewire/portal projects and sometimes it doesn't work well. If you have a trouble with pipewire/portal you must check HyperHDR logs and eventually you can also switch to X11 grabber by deleting all libsmartPipewire.so.* files from /usr/share/hyperhdr/lib folder.

**Reply by epictechlab** [reply]
Thanks for your response,

I´m fairly new to Linux especially Arch Based systems. And until Now I was using X11 instead of Wayland.
After Install Pipewire and Switching to Wayland, HyperHDR works well besides its long Startup Time...
But Wayland give me Some wierd things on KDE, Which is moslty a KDE Problem, not Yours.

I will make some reserches if there is a better solution to using Wayland.

then I can Give you some response if it`s necessery.

**Reply by awawa-dev** [reply]
Yes, the pipewire subsystem initialization could take some time. With introduction of Wayland, I lost my last hope the simple & reliable screen capturing API can be created and unified across most of modern Linux desktop systems ;) Nevertheless, I will be making some improvements to pipewire capturing soon, still rather unrelated to your current problem, rather performance improvements such as lower CPU usage.

**Reply by awawa-dev** [reply]
@epictechlab you can test latest PR that brings the hardware acceleration for the pipewire screen capturing https://github.com/awawa-dev/HyperHDR/pull/556
